.gitignore

```diff
@@ -9,6 +9,11 @@ target/
 *.log
 *.log.*
 
+buildSrc/gradle/
```

Comment:
@parvez-amplicomm Why is this needed? The project never uses these 3 files. I believe this is something in your setup?

Reply:
It's related to my local setup

Reply:
@parvez-amplicomm Please configure your global gitignore file to include those patterns. See [gitignore](https://git-scm.com/docs/gitignore) docs for details.